ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/graphSearch/parameters/DefaultFootstepPlannerParameters.java

```diff
@@ -1,17 +1,91 @@
 package us.ihmc.footstepPlanning.graphSearch.parameters;
 
+import us.ihmc.tools.property.BooleanStoredPropertyKey;
+import us.ihmc.tools.property.DoubleStoredPropertyKey;
+import us.ihmc.tools.property.IntegerStoredPropertyKey;
+import us.ihmc.tools.property.StoredPropertyKeyList;
 import us.ihmc.tools.property.StoredPropertySet;
 
-public class DefaultFootstepPlannerParameters extends StoredPropertySet implements FootstepPlannerParametersBasics
+public class DefaultFootstepPlannerParameters extends StoredPropertySet implements DefaultFootstepPlannerParametersBasics
```

Comment:
I added the keys to this file, I found it confusing to put the parameters in a different file when this file is named DefaultFootStepPlannerParameters